tests/base/test_base.py

```diff
@@ -1,23 +1,65 @@
-import asyncio
-import unittest
+import os
+from pathlib import Path
 
+import yaml
+from aiohttp import web
+from aiohttp.test_utils import AioHTTPTestCase
+
+from app.api.rest_api import RestApi
 from app.service.app_svc import AppService
+from app.service.auth_svc import AuthService
 from app.service.data_svc import DataService
 from app.service.learning_svc import LearningService
 from app.service.planning_svc import PlanningService
 from app.service.rest_svc import RestService
+from app.utility.base_world import BaseWorld
+
 
+class TestBase(AioHTTPTestCase):
+    test_config = """
+        api_key: ADMIN123
+        app.contact.html: /weather
+        app.contact.http: http://127.0.0.1:8888
+        app.contact.tcp: 127.0.0.1:7010
+        app.contact.udp: 127.0.0.1:7011
+        app.contact.websocket: 127.0.0.1:7012
```

Comment:
any reason to not load the conf/default.yml file instead of re-defining it here? i understand it can change--but that seems good for testing/validation.

Reply:
I think that makes sense.  I'll push up a change.

Reply:
So I updated this.  The tradeoff is that now all the plugins get loaded, and the contacts get created, but haven't figured out how to tear down the UDP contact cleanly yet so we get some warnings.

Reply:
Let me see if I can re-do this with pytest-aiohttp and it might be better